go/engine/merkle_audit.go

```diff
@@ -222,12 +223,19 @@ func MerkleAuditRound(m libkb.MetaContext) (err error) {
 		}
 
 		// 3. Generate a random seqno for the starting root in the audit.
-		randomSeqno, err := randSeqno(*firstSeqno, lastSeqno)
-		if err != nil {
-			return err
+		for {
```

Comment:
can you split this into a little function?

Reply:
moved this logic into randSeqno